gpkit/keydict.py

```diff
@@ -170,3 +173,8 @@ def __getitem__(self, key):
     def __setitem__(self, key, value):
         "Assigns the key itself every time."
         KeyDict.__setitem__(self, key, None)
+
+
+class FastKeyDict(KeyDict):
+    "FsatKeyDicts are KeyDicts that don't map keys, only collate arrays"
```

Comment:
As a developer I'm trying to understand when I can get away with using a `FastKeyDict` vs a `KeyDict`, but I'm not completely clear on that from this docstring.